packages/gp2-components/src/organisms/OutputCard.tsx

```diff
@@ -86,6 +93,8 @@ const OutputCard: React.FC<OutputCardProps> = ({
   link,
   mainEntity,
   detailedView = false,
+  showTags = false,
```

Comment:
Do you think we would have a problem to have showTags as true by default? the tags are not appearing in the tag search page in GP2
<img width="1425" alt="Screenshot 2024-03-26 at 10 00 42" src="https://github.com/yldio/asap-hub/assets/16595804/a2821441-30b0-4965-98b2-24ad561f7f51">